app/static/settings/translations.json

```diff
@@ -34,7 +34,13 @@
         "translate": "translate",
         "light": "light",
         "dark": "dark",
-        "system": "system"
+        "system": "system",
+        "all": "All",
+        "images": "Images",
+        "maps": "Maps",
+        "videos": "Videos",
+        "news": "News",
+        "books": "Books"
```

Comment:
Equal translations are enforced in the tests, so these have to be translated to the existing languages. I'd like to automate this in the future, but haven't looked into it yet. 

That being said, I can take care of these. Whenever the feature is ready to merge, I'll go through and add the remaining translations before merging.

Reply:
Yes, I saw that the test didn't pass due to the keys not being the same on all languages, but I have just filled the English to have the "reference" keys.